README.md

```diff
@@ -86,3 +86,4 @@ Copyright © 2022 [Will Jaw](https://github.com/jjaw).<br />
 This project is [Apache-2.0](LICENSE) licensed.
 
 ---
+
```

Comment:
Thanks! I am happy to see a PR after a long time in CityDAO repo:) 
Approving to save the day but need to automate this. maybe:
https://dev.to/gautamkrishnar/how-to-prevent-github-from-suspending-your-cronjob-based-triggers-knf